src/SDKs/CognitiveServices/dataPlane/Vision/Vision/Generated/Face/Models/SimilarFace.cs

```diff
@@ -55,36 +57,32 @@ public SimilarFaceResult(string faceId = default(string), string persistedFaceId
         /// detection call
         /// </summary>
         [JsonProperty(PropertyName = "faceId")]
-        public string FaceId { get; set; }
+        public System.Guid? FaceId { get; set; }
 
         /// <summary>
         /// Gets or sets persistedFaceId of candidate face when find by
         /// faceListId. persistedFaceId in face list is persisted and will not
         /// expire. As showed in below response
         /// </summary>
         [JsonProperty(PropertyName = "persistedFaceId")]
-        public string PersistedFaceId { get; set; }
+        public System.Guid? PersistedFaceId { get; set; }
```

Comment:
nullable?

Reply:
Same as the above comment. 
And if we don't use method overload, PersistedFaceId is nullable because faceId will return when the FindSimilarRequest's target are faceIds.

Reply:
OK, this convince me well. Let's keep it currently.